packages/components/src/ErrorPage/subcomponents/BaseErrorPage.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { HTMLAttributes } from "react"
+import { FormattedMessage, useIntl } from "@cultureamp/i18n-react-intl"
+import classNames from "classnames"
+import { BrandMoment } from "@kaizen/brand-moment"
+import arrowRightIcon from "@kaizen/component-library/icons/arrow-right.icon.svg"
+import email from "@kaizen/component-library/icons/email.icon.svg"
+import { BrandMomentError } from "@kaizen/draft-illustration"
+import { Paragraph } from "@kaizen/typography"
+import { OverrideClassName } from "~types/OverrideClassName"
+import styles from "./BaseErrorPage.module.scss"
+
+const getMailToHref = (code: number): string => {
+  const supportEmail = "support@cultureamp.com"
+  const subject = "Houston we have a problem"
+  const body = `Hi there,\n\nI received a ${code} error page while I was trying to...`
+  return encodeURI(`mailto:${supportEmail}?subject=${subject}&amp;body=${body}`)
+}
+
+const HOME_HREF = "/app/home"
+
+export type BaseErrorPageProps = {
+  code: number
+  title: string
+  message: React.ReactNode | string
+  callToAction?: {
+    onContactSupport: () => void
+    homeHref?: string
+  }
+} & OverrideClassName<HTMLAttributes<HTMLDivElement>>
+
+export const BaseErrorPage = ({
```

Comment:
Looking at the props for this `Base` it doesn't look like there's much room for customisation if the consumers were to use this directly, only being able to alter the text of two props + `callToAction`.
Also each Error Page doesn't seem to need to be customised further than updating the `title` and `message`, which leads me into thinking that this multi-component approach is a bit too verbose.

What if we consider the `title` and `message` as optional props with defaults that get filled in based on the code. So something like:

```
const ErrorPage = ({ code, message }) => {
  const messages = { 
    404: " ... "
  }

  return (<> {message ?? messages[code]} </>
}
```
This to me achieves basically the same amount of customisation and preset defaults as it does now without needing multiple individual components. We also won't need to create a component for every new error code, just add some new messages to the object.

Reply:
> Looking at the props for this Base it doesn't look like there's much room for customisation
You can change all the internals of what this component does with these props so I don't know what you mean here.

> What if we consider the title and message as optional props with defaults that get filled in based on the code. So something like

Sure it could be done like that.